src/transformers/models/layoutlmv2/feature_extraction_layoutlmv2.py

```diff
@@ -220,3 +233,54 @@ def __call__(
             encoded_inputs["boxes"] = boxes_batch
 
         return encoded_inputs
+
+    def post_process(self, outputs, target_sizes, offset_mapping, bbox):
+        """
+        Converts the output of :class:`~transformers.LayoutLMv2ForTokenClassification` into the format expected by the COCO api.
+        Only supports PyTorch.
+
+        Args:
+            outputs :obj:`Dict`):
+                Raw outputs of the model.
+            target_sizes (:obj:`torch.Tensor` of shape :obj:`(batch_size, 2)`):
+                Tensor containing the size (h, w) of each image of the batch. For evaluation, this must be the original
+                image size (before any data augmentation). For visualization, this should be the image size after data
+                augment, but before padding.
```

Comment:
```suggestion
                Tensor containing the original size (h, w) of each image of the batch. 
```
LayoutLMv2 doesn't require any data augmentation, so we can leave that out.